src/styles/index.scss

```diff
@@ -1,3 +1,129 @@
 body {
-  margin: 0;
+  $card-border: #f3f3f3;
+  $color-black: #060b35;
+  $color-gray: #616070;
+  $color-blue: #00acdc;
+  $color-white: #fff;
+  $gap: 4px;
+
+  body,
+  p,
+  a,
+  img,
+  div {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Roboto, Helvetica, sans-serif;
+  }
+
+  .card {
```

Comment:
why do we need these styles if we have them in cars.scss and stars.scss, we can just import those files here